### DISCUSSION TITLE
Adding more metadata to PdfTemplate examples

### DISCUSSION THREAD

**Discussion by jose-guevarra** [oss-review-toolkit/ort#8564] [General]
Hi,  does anyone have any examples of modifying the PdfTemplate(AsciiDoc) to add more package metadata? In particular, authors and description.  It would be nice to have a table with package name, version, license as well.

**Comment by sschuberth** [comment]
In general, any FreeMarker-based reporter has access to all these top-level variables:

https://github.com/oss-review-toolkit/ort/blob/4e73f5af19e2270e21545f77ef61f1a4a54b17f0/plugins/reporters/freemarker/src/main/kotlin/FreemarkerTemplateProcessor.kt#L412-L421

Unfortunately, `packages` is of type `FreemarkerTemplateProcessor.PackageModel` which does not include all package metadata, like the authors or the description. However, `ortResult.getPackages()` gives you `CuratedPackage`s whose `metadata` contains all details.
